packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -149,6 +170,7 @@ export const Collection = <T,>(props: CollectionProps<T>) => {
             key={index}
             disabled={disabled}
             draggable={draggable}
+            error={error}
```

Comment:
The error should probably be rendered underneath the whole collection. We don't need to manage the individual item's errors here.

Reply:
hmmm ok I thought for the experience it would be better to put it under the faulty input no?

Reply:
Right now your code doesn't do anything with the error 😅